apps/server/src/modules/account/services/account.service.integration.spec.ts

```diff
@@ -104,6 +105,12 @@ describe('AccountService Integration', () => {
 					provide: LegacyLogger,
 					useValue: createMock<LegacyLogger>(),
 				},
+				{
+					provide: EventBus,
+					useValue: {
```

Comment:
createMock<EventBus>() ? But why not createMock from CqrsModule and this as provider?

Reply:
@CeEv could you clarify it?